components/applications-service/pkg/storage/postgres/service_groups.go

```diff
@@ -21,115 +21,74 @@ import (
-`
+	// 3) Provides an array of all service releases within a service group
+	// 4) Determines the overall status of the service group
 
 	// TODO: Update this query once we understand better the deploying status
 	selectServiceGroupsHealthCounts = `
-SELECT COUNT(*) AS total
-  , COUNT(*) FILTER (
-      WHERE health_critical > 0
-    ) AS critical
-  , COUNT(*) FILTER (
-      WHERE health_unknown  > 0
-        AND health_critical = 0
-    ) AS unknown
-  , COUNT(*) FILTER (
-      WHERE health_warning  > 0
-        AND health_critical = 0
-        AND health_unknown  = 0
-    ) AS warning
-  , COUNT(*) FILTER (
-      WHERE health_ok > 0
-        AND health_critical = 0
-        AND health_warning  = 0
-        AND health_unknown  = 0
-    ) AS ok
-FROM (` + selectServiceGroupHealth + `) AS service_groups_health_counts
+	SELECT COUNT(*) AS total
+	,COUNT(*) FILTER (
+						 WHERE health_critical > 0
+			 ) AS critical
+	,COUNT(*) FILTER (
+						 WHERE health_unknown  > 0
+							 AND health_critical = 0
+			) AS unknown
+	,COUNT(*) FILTER (
+						 WHERE health_warning  > 0
+							 AND health_critical = 0
+							 AND health_unknown  = 0
+			 ) AS warning
+	,COUNT(*) FILTER (
+						 WHERE health_ok > 0
+							 AND health_critical = 0
+							 AND health_warning  = 0
+							 AND health_unknown  = 0
+			 ) AS ok
+FROM service_groups AS service_groups_health_counts
 `
 
 	selectServiceGroupHealthWithPageSort = `
-SELECT * FROM (` + selectServiceGroupHealth + `) AS service_groups_health
-ORDER BY %s
-LIMIT $1
+SELECT * FROM service_groups AS service_groups_health
+ ORDER BY %s
+ LIMIT $1
 OFFSET $2
 `
 	selectServiceGroupHealthFilterCRITICAL = `
-SELECT * FROM (` + selectServiceGroupHealth + `) AS service_groups_health
-WHERE health_critical > 0
-ORDER BY %s
-LIMIT $1
+SELECT * FROM service_groups AS service_groups_health
+ WHERE health_critical > 0
+ ORDER BY %s
+ LIMIT $1
 OFFSET $2
 `
 	selectServiceGroupHealthFilterUNKNOWN = `
-SELECT * FROM (` + selectServiceGroupHealth + `) AS service_groups_health
-WHERE health_unknown  > 0
-  AND health_critical = 0
-ORDER BY %s
-LIMIT $1
+SELECT * FROM service_groups AS service_groups_health
+ WHERE health_unknown  > 0
+   AND health_critical = 0
+ ORDER BY %s
+ LIMIT $1
 OFFSET $2
 `
 	selectServiceGroupHealthFilterWARNING = `
-SELECT * FROM (` + selectServiceGroupHealth + `) AS service_groups_health
-WHERE health_warning  > 0
-  AND health_critical = 0
-  AND health_unknown  = 0
-ORDER BY %s
-LIMIT $1
+SELECT * FROM service_groups AS service_groups_health
+ WHERE health_warning  > 0
+   AND health_critical = 0
+   AND health_unknown  = 0
+ ORDER BY %s
+ LIMIT $1
 OFFSET $2
 `
 	selectServiceGroupHealthFilterOK = `
-SELECT * FROM (` + selectServiceGroupHealth + `) AS service_groups_health
-WHERE health_ok > 0
-  AND health_critical = 0
-  AND health_warning  = 0
-  AND health_unknown  = 0
-ORDER BY %s
-LIMIT $1
+SELECT * FROM service_groups AS service_groups_health
```

Comment:
You forgot to update all the queries with the new view name. (`s/service_groups/service_group_health/g`)
```suggestion
SELECT * FROM service_group_health AS service_group_health
```

Also, we don't need the `AS` statement here anymore and it is not matching the new view name since it is plural, so perhaps you could remove it or update it as above.